test/connection.js

```diff
@@ -199,7 +270,11 @@ describe('Hapi server', () => {
         }, (err) => {
 
             expect(err).to.not.exist();
-            expect(server.plugins['hapi-mongodb'].db.options.url).to.equal('mongodb://localhost:27017');
+            const db = server.plugins['hapi-mongodb'].db;
+            expect(db).to.be.instanceof(Mongodb.Db);
+            expect(db.databaseName).to.equal('test');
+            expect(db.topology.s.clonedOptions.host).to.equal('localhost');
```

Comment:
This seems to be relying on mongodb driver internals, don't we have better than that ?